corehq/apps/smsbillables/management/commands/add_sms_gateway_permissions.py

```diff
@@ -61,10 +61,10 @@ def handle(self, usernames, **options):
                         from_role=user_role.role,
                         to_role=global_sms_gateway_access,
                     )
-                    print("Enabled privilege to Edit Global SMS Gateways for the user {user.username}")
+                    print(f"Enabled privilege to Edit Global SMS Gateways for the user {username}")
                 else:
                     print("User %s already have the requested privilege"
                           % user.username)
```

Comment:
shouldn't this also just be `username` instead of `user.username`?

Reply:
We have a user object with us that has a username attribute. I guess it should work fine.

Reply:
For consistency, I have updated it to be `user.username` at both the places https://github.com/dimagi/commcare-hq/pull/32115/commits/7c4d4ee787b1011b103b169b6d4d3b030446daa9

Reply:
that makes sense. Thanks!